qiskit/chemistry/drivers/fcidumpd/fcidumpdriver.py

```diff
@@ -0,0 +1,104 @@
+# -*- coding: utf-8 -*-
+
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2020.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+
+""" FCIDump Driver """
+
+from qiskit.chemistry.drivers import BaseDriver
+from qiskit.chemistry import QiskitChemistryError, QMolecule
+from qiskit.chemistry.drivers.fcidumpd.dumper import dump
+from qiskit.chemistry.drivers.fcidumpd.parser import parse
+
+
+class FCIDumpDriver(BaseDriver):
+    """
+    Python implementation of an FCIDump driver.
+
+    The FCIDump format is partially defined in Knowles1989.
+
+    Knowles1989:
+        Peter J. Knowles, Nicholas C. Handy,
+        A determinant based full configuration interaction program,
+        Computer Physics Communications, Volume 54, Issue 1, 1989, Pages 75-83,
+        ISSN 0010-4655, https://doi.org/10.1016/0010-4655(89)90033-7.
+    """
```

Comment:
I think this should be:
```suggestion
    References:
        Knowles1989: Peter J. Knowles, Nicholas C. Handy,
            A determinant based full configuration interaction program,
            Computer Physics Communications, Volume 54, Issue 1, 1989, Pages 75-83,
            ISSN 0010-4655, https://doi.org/10.1016/0010-4655(89)90033-7.
    """
```